src/esbmc/options.cpp

```diff
@@ -435,7 +435,9 @@ const struct group_opt_templ all_cmd_options[] = {
     // Abort if the program contains a recursion
     {"abort-on-recursion", NULL, ""},
     // Verbosity of message, probably does nothing
```

Comment:
Shall we update the comment? "probably does nothing" -> "see PR128 for fine-grained control over debug log message"